src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1058,7 +1058,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 $useTargetObject = $true
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-windows`
Tests for $ErrorView.ConciseView tests.Script error should contain path to script and line for error
```powershell
Expected like wildcard '*C:\Users\VssAdministrator\AppData\Local\Temp\b9e74acc-eeca-4e56-83a8-887f0637dd03\test.ps1:4' to match '
ParserError: C:\Users\VssAdministrator\AppData\Local\Temp\b9e74acc-eeca-4e56-83a8-887f0637dd03\test.ps1:4
Line |
   4 |                 123)

     |                    ^ Unexpected token ')' in expression or statement.

', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Formatting\ErrorView.Tests.ps1: line 35
35:             $e | Out-String | Should -BeLike "*${testScriptPath}:4"
```
Enter-PSHostProcess tests.By Process Id.Can enter, exit, and re-enter another Windows PowerShell PSHost
```powershell
Expected $true, because The script was able to re-enter another process and grab the pid of '1132'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 120
120:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pid of '$powershellId'."
```